### PR TITLE
Dynalite Integration Documentation

### PR DESCRIPTION
**Pull request by ziv1234**
**Description:**

I created a new integration for Dynalite devices (home assistant pull request 27841), and this is the matching documentation.

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#27841

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/dynalite.markdown [hunk 1]
```diff
@@ -0,0 +1,140 @@
+---
+title: "Philips Dynalite"
+description: "Instructions on setting up Philips Dynalite within Home Assistant."
+logo: dynalite.png
+ha_category:
+  - Hub
+  - Light
+ha_iot_class: Local Push
+ha_qa_scale: No Score
+featured: true
```
Comment (klaasnicolaas) [inline source/_integrations/dynalite.markdown]: Please remove `featured`

File: source/_integrations/dynalite.markdown [hunk 2]
```diff
@@ -0,0 +1,140 @@
+---
+title: "Philips Dynalite"
+description: "Instructions on setting up Philips Dynalite within Home Assistant."
+logo: dynalite.png
+ha_category:
+  - Hub
+  - Light
+ha_iot_class: Local Push
+ha_qa_scale: No Score
```
Comment (klaasnicolaas) [inline source/_integrations/dynalite.markdown]: If there is no score then leave it away

File: source/_integrations/dynalite.markdown [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+featured: true
+ha_release: "0.60"
```
Comment (klaasnicolaas) [inline source/_integrations/dynalite.markdown]:
```suggestion
ha_release: 0.101
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_integrations/dynalite.markdown (hunk 3 above)
Comment (ziv1234) [reply source/_integrations/dynalite.markdown]: thanks. will make these changes now. i used hue as a base as there are many similarities, but didn't notice. will change and push now

Comment (ziv1234) [reply source/_integrations/dynalite.markdown]: done. uploaded to fork

File: source/_integrations/dynalite.markdown (hunk 1 above)
Comment (ziv1234) [reply source/_integrations/dynalite.markdown]: done

File: source/_integrations/dynalite.markdown (hunk 2 above)
Comment (ziv1234) [reply source/_integrations/dynalite.markdown]: done. thanks

File: source/_integrations/dynalite.markdown (hunk 3 above)
Comment (ziv1234) [reply source/_integrations/dynalite.markdown]: done

**Comment by ziv1234** [comment]
Hello. Is there anything else I need to do here? Please let me know

**Comment by frenck** [comment]
This PR awaits a review and the parent PR needs processing. It is a busy season, please have some patience. Thanks 👍

**Pull request merged**